test/basic.html

```diff
@@ -88,6 +88,37 @@
         });
       });
 
+      suite('focus/blur events', function() {
```

Comment:
If `IronControlState` implements and tests this behavior, is it useful for `iron-autogrow-textarea` to also test the behavior?